tests/req_mirror.php

```diff
@@ -17,6 +17,13 @@
  */
 
 $body = file_get_contents('php://input');
+
+/** responseDelay url paramter define delay of response.(millisecond) */
+$delayMs = isset($_GET["responseDelay"]) ? $_GET["responseDelay"] : null;
+if (!is_null($delayMs)) {
+    usleep($delayMs * 1000);
```

Comment:
You can directly use the variable.
```suggestion
if (isset($_GET["responseDelay"]) && is_numeric($_GET["responseDelay"])) {
    usleep($_GET["responseDelay"]) * 1000);
```

Reply:
Thank you for the suggesting.